src/metaswitch/clearwater/cluster_manager/main.py

```diff
@@ -10,42 +10,49 @@
 """Clearwater Cluster Manager
 
 Usage:
-  main.py --mgmt-local-ip=IP --sig-local-ip=IP --local-site=NAME --remote-site=NAME --remote-cassandra-seeds=IPs --uuid=UUID --etcd-key=KEY --etcd-cluster-key=CLUSTER_KEY
-          [--signaling-namespace=NAME] [--foreground] [--log-level=LVL]
-          [--log-directory=DIR] [--pidfile=FILE] [--cluster-manager-enabled=Y/N]
+  main.py --mgmt-local-ip=IP --sig-local-ip=IP --local-site=NAME --uuid=UUID (--etcd-key=KEY | --db-key=KEY) (--etcd-cluster-key=CLUSTER_KEY | --db-cluster-key=CLUSTER_KEY)
+          [--remote-site=NAME] [--remote-cassandra-seeds=IPs] [--cassandra-container-id=ID] [--signaling-namespace=NAME] [--foreground] [--log-level=LVL]
+          [--log-directory=DIR] [--pidfile=FILE] [--cluster-manager-enabled=Y/N] [--etcd | --consul]
 
 Options:
   -h --help                      Show this screen.
-  --mgmt-local-ip=IP             Management IP address
-  --sig-local-ip=IP              Signaling IP address
+  --mgmt-local-ip=IP             Management IP address (for the back-end)
+  --sig-local-ip=IP              Signaling IP address (must be unique for each node in the cluster)
   --local-site=NAME              Name of local site
-  --remote-site=NAME             Name of remote site
-  --remote-cassandra-seeds=IPs   Comma separated list of at least one IP address from each remote Cassandra site
-  --signaling-namespace=NAME     Name of the signaling namespace
   --uuid=UUID                    UUID uniquely identifying this node
   --etcd-key=KEY                 Etcd key (top level)
+  --db-key=KEY                   Top-level key in back-end database
   --etcd-cluster-key=CLUSTER_KEY Etcd key (used in the data store clusters)
+  --db-cluster-key=CLUSTER_KEY   Back-end database key (used in the data store clusters)
+  --remote-site=NAME             Name of remote site
+  --remote-cassandra-seeds=IPs   Comma separated list of at least one IP address from each remote Cassandra site
+  --cassandra-container-id=ID    When containerised, the ID of the local Cassandra container
+  --signaling-namespace=NAME     Name of the signaling namespace
   --foreground                   Don't daemonise
   --log-level=LVL                Level to log at, 0-4 [default: 3]
   --log-directory=DIR            Directory to log to [default: ./]
   --pidfile=FILE                 Pidfile to write [default: ./cluster-manager.pid]
   --cluster-manager-enabled=Y/N  Whether the cluster manager should start any threads [default: Yes]
+  --etcd                         Use the etcd back-end (the default)
+  --consul                       Use the Consul back-end
```

Comment:
should this be --backend [etcd|consul]?  Might avoid worries about both being set?